style.css

```diff
@@ -0,0 +1,252 @@
+.container {
+  position: relative;
+  width: 100%;
+  margin: 50px;
+  max-width: 1000px;
+  min-height: 1000px;
+
+  background: white;
+  box-shadow: 20px 20px 15px 0px #55aead;
+
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+}
+
+.left_side {
+  position: relative;
+  padding: 40px;
+  background: #003147;
+}
+
+.prfileText {
+  position: relative;
+  display: flex;
+
+  flex-direction: column;
+  align-items: center;
+
+  padding-bottom: 20px;
+  border-bottom: 1px solid rgba(255, 255, 255, 0.2);
+}
+
+.imgBx {
+  position: relative;
+  width: 200px;
+  height: 200px;
+
+  border-radius: 50%;
+  overflow: hidden;
+}
+
+.imgBx img {
+  position: absolute;
+  inset: 0;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.prfileText h2 {
+  color: #fff;
+  font-size: 1.3em;
+  text-align: center;
+  text-transform: uppercase;
+  font-weight: 600;
+  line-height: 1.4em;
+}
+
+.prfileText h2 span {
+  font-size: 0.8em;
+  font-weight: 300;
+}
+
+.contactInfo {
+  padding-top: 40px;
+}
+.title {
+  color: #fff;
+  font-weight: 600;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+}
+.contactInfo ul {
+  position: relative;
+}
+.contactInfo ul li {
+  position: relative;
+  margin: 10px 0;
+
+  list-style: none;
+  cursor: pointer;
+}
+.icon {
+  display: inline-block;
+  width: 30px;
+
+  font-size: 18px;
+  color: #03a9fa;
+}
+.text {
+  color: #fff;
+  font-weight: 300;
+}
+
+.contactInfo.education li {
+  margin-bottom: 15px;
+}
+.contactInfo.education h5 {
+  color: #03a9fa;
+  font-weight: 500;
+}
+.contactInfo.education h4:nth-child(2) {
+  color: #fff;
+  font-weight: 400;
+}
+.contactInfo.education h4 {
+  color: #fff;
+  font-weight: 300;
+}
+
+.contactInfo.language .percent {
+  position: relative;
+  width: 100%;
+  height: 6px;
+  display: block;
+
+  background: #081921;
+  margin-top: 5px;
+}
+.contactInfo.language .percent div {
+  position: absolute;
+  inset: 0;
+  height: 100%;
+  background: #03a9fa;
+}
+
+.right_side {
+  position: relative;
+  padding: 40px;
+  background: #fff;
+}
+.about {
+  margin-bottom: 50px;
+}
+.about:last-child {
+  margin-bottom: 0;
+}
+.title2 {
+  color: #003147;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+
+  margin-bottom: 10px;
+}
+p {
+  color: #333;
+}
+.about .box {
+  display: flex;
+  flex-direction: row;
+
+  margin: 20px 0;
+}
+.about .box .year_company {
+  min-width: 150px;
+}
+.about .box .year_company h5 {
+  text-transform: uppercase;
+  color: #848c90;
+}
+.about .box .text h4 {
+  text-transform: uppercase;
+  color: #03a9fa;
+  font-size: 16px;
+}
+
+.skills .box {
+  position: relative;
+  width: 100%;
+
+  display: grid;
+  grid-template-columns: 150px 1fr;
+  justify-content: center;
+  align-items: center;
+}
+.skills .box h4 {
+  text-transform: uppercase;
+  color: #848c90;
+  font-weight: 400;
+}
+.skills .box .percent {
+  position: relative;
+  width: 100%;
+  height: 10px;
+
+  background-color: #f0f0f0;
+}
+.skills .box .percent div {
```

Comment:
avoid using lengthy selectors

Reply:
for understanding purposes only I write like this